### PR TITLE
Add optional param sourceId to content share connectivity check

### PR DESCRIPTION
**Pull request by anuranduttaroy**
**Issue #:** NA

**Description of changes:**
Add optional param sourceId to content share connectivity check for electron apps

**Testing**

1. Have you successfully run `npm run build:release` locally?
Yes.

2. How did you test these changes?
Tested manually and ran integration tests successfully.

3. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
Yes

4. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
NA

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/meetingreadinesschecker/DefaultMeetingReadinessChecker.ts [hunk 1]
```diff
@@ -206,7 +206,9 @@ export default class DefaultMeetingReadinessChecker implements MeetingReadinessC
     return trackConstraints;
   }
 
-  async checkContentShareConnectivity(): Promise<CheckContentShareConnectivityFeedback> {
+  async checkContentShareConnectivity(
+    sourceId: string = null
```
Comment (ltrung) [inline src/meetingreadinesschecker/DefaultMeetingReadinessChecker.ts:210]: Could we do sourceId? similar to startContentShareScreenCapture

File: test/dommock/DOMMockBuilder.ts [hunk 2]
```diff
@@ -347,7 +347,9 @@ export default class DOMMockBuilder {
               // @ts-ignore
               mediaStreamTrack.kind = 'video';
             }
-            mediaStream.addTrack(mediaStreamTrack);
+            if (!mockBehavior.getDisplayMediaWithSourceId) {
+              mediaStream.addTrack(mediaStreamTrack);
```
Comment (ltrung) [inline test/dommock/DOMMockBuilder.ts:351]: Why do we need to do this? mediaStream from getUserMedia should have tracks.

File: src/meetingreadinesschecker/DefaultMeetingReadinessChecker.ts (hunk 1 above)
Comment (anuranduttaroy) [reply src/meetingreadinesschecker/DefaultMeetingReadinessChecker.ts:210]: We are doing that in [MeetingReadinessChecker.ts](https://github.com/aws/amazon-chime-sdk-js/pull/687/commits/61a79a42f88fe7e900932dc4ff8fce3f5009be0b#diff-9d1c496522a94ecd712beccbc3e7d25bR47). In the implementation, we are setting it to `null` by default.

File: test/dommock/DOMMockBuilder.ts (hunk 2 above)
Comment (anuranduttaroy) [reply test/dommock/DOMMockBuilder.ts:351]: Synced with you offline. Fixing the DOMMockBuilder for this.

**Review by ltrung — APPROVED** [review]

**Pull request merged**